README.md

```diff
@@ -149,6 +149,18 @@ Grab the values for the `AWS_ACCESS_KEY_ID` and the `AWS_SECRET_ACCESS_KEY` and
 
 Now you should be able to start uploading some files!
 
+## Review Apps
+This repo is set up to deploy review apps on Heroku, and those pull from the staging database to match the experience of deploying as closely as possible! However, note that in order to prevent unapproved model changes from effecting the staging database, migrations are prevented from running on review apps. So those will still have to be reviewed locally.
+
+### Review apps for Wagtail changes
+
+1. Configure the S3 connection, as documented above.
+2. Once a release phase successfully completes, provision an Essential 0 database for your review app.
+3. Retrieve the URI from the Credentials tab.
+4. Run `export HEROKU_DATABASE_URL=<YOUR URI HERE>`.
+5. If needed, run `docker compose run --rm -e DATABASE_URL=${HEROKU_DATABASE_URI} scrapers` to populate initial legislative data.
+6. Run `docker compose run --rm -e DATABASE_URL=${HEROKU_DATABASE_URI} app python manage.py load_content` to populate initial CMS content from the version controlled fixtures.
```

Comment:
If we need to test Wagtail stuff, we need a writeable database for review apps. This is how you make one and populate it with enough initial data to be dangerous.